### PR TITLE
Implement select palette

### PR DESCRIPTION
**Pull request by VictorAlbertos**
| Collapsed  | Expanded | Ocean | Fresh
| ------------- | ------------- | ------------- | ------------- |
|   |  |  |

### CODE REVIEW

File: showcase/src/main/kotlin/com/recco/showcase/main/PaletteSelection.kt [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        transitionSpec = { tween(500) },
+        label = ""
+    ) { state ->
+        if (state) value else 0f
+    }
+
+    return anim
+}
+
+fun Modifier.noRippleClickable(onClick: () -> Unit): Modifier = composed {
```
Comment (chemarubio) [inline showcase/src/main/kotlin/com/recco/showcase/main/PaletteSelection.kt:293]: This extension function is already included in com.recco.internal.core.ui.extensions, maybe we want it to be imported from there ?

File: showcase/src/main/kotlin/com/recco/showcase/main/MainScreen.kt [hunk 2]
```diff
@@ -38,94 +44,83 @@ fun MainScreen(
...
-            border = BorderStroke(width = 1.dp, WarmBrown),
-            onClick = logoutClick
-        ) {
-            Text(
-                text = stringResource(id = R.string.logout),
-                style = Typography.displayMedium.copy(color = WarmBrown)
-            )
-        }
+        val application = LocalContext.current.applicationContext as Application
+        PaletteSelection(selectionClickPalette = { reccoPalette ->
```
Comment (chemarubio) [inline showcase/src/main/kotlin/com/recco/showcase/main/MainScreen.kt:121]: Does it make sense to hold a state in which default palette is referenced ? With the current design, at least IMHO it is not clear which is the current selected palette.

**Review by chemarubio — APPROVED** [review]
🚀

File: showcase/src/main/kotlin/com/recco/showcase/main/PaletteSelection.kt (hunk 1 above)
Comment (VictorAlbertos) [reply showcase/src/main/kotlin/com/recco/showcase/main/PaletteSelection.kt:293]: I don't want to make a bad of example of consuming our own SDK xD Everything that it is inside an internal module, should not consumed by clients ;)

File: showcase/src/main/kotlin/com/recco/showcase/main/MainScreen.kt (hunk 2 above)
Comment (VictorAlbertos) [reply showcase/src/main/kotlin/com/recco/showcase/main/MainScreen.kt:121]: Let's discuss it during the call, I did not find other way to achieve this.

**Pull request merged**